roles/usermanager/defaults/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 
+install_keys_to_service_user: false
+key_name: "id_rsa.pub"
```

Comment:
Are these two parameters `key_name` and `install_keys_to_service_user` related to terraform?